src/styles/main.css

```diff
@@ -30,6 +30,11 @@
 @tailwind base;
 
 /* Write your own custom base styles here */
+#__next {
```

Comment:
@ynotdraw Wasn't quite sure this was the right place to style this?

Reply:
Yup I do the same thing but with:

```
@media (min-width: 768px) {
  #__next {
    min-height: 100vh;
    display: grid;
    grid-template-rows: auto 1fr auto;
  }
}
```